package.json

```diff
@@ -75,6 +75,7 @@
     "cpx": "~1.5.0",
     "execa": "0.8.0",
     "husky": "^0.14.3",
+    "jsonschema": "1.2.4",
```

Comment:
Have we considered using ajv? (https://github.com/epoberezkin/ajv), it includes the json meta schema's themselves so we wouldn't have to include them in each project